File: GoEnv/go/common/EdocDistCert/Test/BlabEaddrGetCanceledCompanyClientTest.go
package main

import (
	blabClient "../Client"
	testClient "./TestConfig"
	"log"
)

// 공인전자주소 탈퇴이력 조회 테스트(법인)
func main() {
	log.SetFlags(log.LstdFlags | log.Lshortfile)
	err := testClient.BlabTestClientInit()
	if err != nil {
		return
	}

	clientConfig := testClient.ClientConfig
	appRes, err := blabClient.GetEaddrGetCanceled(clientConfig.Server.BaseUrl, clientConfig.Company.Idn)
	if err != nil {
		log.Printf("err=%+v\n", err)
		return
	}

	log.Printf("appRes=%+v", appRes)
	log.Printf("appRes.Data=%+v", appRes.Data)
}

File: GoEnv/go/common/EdocDistCert/Test/node/BlabEaddrGetCanceledCompanyClientTest.mjs
import { v4 } from 'uuid';
import dateFormat, {masks} from 'dateformat';
import http from 'http';
import blabClient from './BlabClient.mjs';
import clientConfig from './BlabClientConfig.mjs';

// 법인 공인전자주소 탈퇴이력 조회 테스트
function BlabEaddrGetCanceledCompanyTest() {
    blabClient.GetEaddrCanceled(
        clientConfig.server.baseUrl,
        clientConfig.company.idn)
        .then(res => {
            console.log(res)
        });
}

BlabEaddrGetCanceledCompanyTest();